### PR TITLE
Tag style update

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Addresses https://github.com/orgs/pysat/projects/4/views/1 by fixing the Instrument method kwarg defaults.  Also improved the docstrings.

# Type of change

Please delete options that are not relevant.

- Style update

# How Has This Been Tested?

Python broken locally, relying on CI.

## Test Configuration
* Operating system: GitHub Actions
* Version number: Python 3.7-10
* Any details about your local setup that are relevant: broken 😿 

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have linted the files updated in this pull request
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: pysatMadrigal/instruments/gnss_tec.py [hunk 1]
```diff
@@ -77,18 +77,12 @@
 
 
 def init(self):
-    """Initializes the Instrument object with values specific to GNSS TEC
+    """Initializes the Instrument object with values specific to GNSS TEC."""
```
Comment (aburrell) [inline pysatMadrigal/instruments/gnss_tec.py:80]:
```suggestion
    """Initialize the Instrument object with values specific to GNSS TEC."""
```

File: pysatMadrigal/instruments/gnss_tec.py [hunk 2]
```diff
@@ -174,19 +168,19 @@ def download(date_array, tag='', inst_id='', data_path=None, user=None,
     return
 
 
-def load(fnames, tag=None, inst_id=None):
-    """ Routine to load the GNSS TEC data
+def load(fnames, tag='', inst_id=''):
+    """Routine to load the GNSS TEC data.
```
Comment (aburrell) [inline pysatMadrigal/instruments/gnss_tec.py:172]:
```suggestion
    """Load the GNSS TEC data.
```

File: pysatMadrigal/instruments/jro_isr.py [hunk 3]
```diff
@@ -217,19 +215,19 @@ def download(date_array, tag='', inst_id='', data_path=None, user=None,
                      user=user, password=password, file_type=file_type)
 
 
-def load(fnames, tag=None, inst_id=None):
+def load(fnames, tag='', inst_id=''):
     """ Routine to load the JRO ISR data
```
Comment (aburrell) [inline pysatMadrigal/instruments/jro_isr.py:219]:
```suggestion
    """Load the JRO ISR data.
```

**Comment by rstoneback** [comment]
Re-running tests

File: pysatMadrigal/instruments/gnss_tec.py [hunk 4]
```diff
@@ -174,19 +168,19 @@ def download(date_array, tag='', inst_id='', data_path=None, user=None,
...
-    """ Routine to load the GNSS TEC data
+def load(fnames, tag='', inst_id=''):
+    """Load the GNSS TEC data.
 
     Parameters
     ----------
     fnames : list
         List of filenames
-    tag : str or NoneType
+    tag : st
```
Comment (rstoneback) [inline pysatMadrigal/instruments/gnss_tec.py:178]:
```suggestion
    tag : str
```

**Comment by rstoneback** [comment]
One of the tests was cancelled. Checking if this was intentional or perhaps the system being wonky.

**Comment by rstoneback** [comment]
The error is now ```unable to open url http://cedar.openmadrigal.org/```.

**Comment by aburrell** [comment]
It looks like enough of the versions are passing to allow merging with the remote server failures if you are ok to approve.  Your call.

**Comment by rstoneback** [comment]
Fine with me. I'll let this round do its thing since already running.

**Review by rstoneback — APPROVED** [review]
Thanks for the standards tweak!

**Pull request merged**